.github/workflows/builddoc.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         "https://astral.sh/uv/install.sh"
         | sh
     - name: Install dependencies
-      run: uv pip install .[docs]
+      run: uv pip install --group docs
```

Comment:
All the workflows seem to be failing with `error: unexpected argument '--group' found` -- `uv` may not have implemented support for dependency groups with the `uv pip install` command yet...

Reply:
It does not appear to be yet supported by pip itself? https://github.com/pypa/pip/issues/12963

Reply:
Ah, for some reason I thought `pip` had already merged support. Let's leave this PR as draft until `pip` gets support, then.

Reply:
Yeah we don't have support in the pip interface yet because it depends what pip does :)